### ISSUE TITLE
Map doesn't center on marker

### ISSUE THREAD

**Issue by GabrielCW** [binhums/ember-cli-mapbox#41]
Even when using the recenter option, on a single map marker, the map doesn't center on it (or on the coordinates I entered in center). 
I suspect it has to do with map height, because whatever the height of the div, the marker is always the same distance from the top, but I couldn't find any option to set the map dimensions.

How could I troubleshoot this/get the map centered?

Here's the template:
```hbs
<div class="row" style="height:200px;background-color:#eee;overflow: hidden;">
    {{#mapbox-map
        mapId='my_map_id'
        center=(mapbox-coords contact.address.latitude contact.address.longitude)
        zoom=13 as |map|}}
            {{mapbox-marker map=map coordinates=contact.address.coordinates recenter=true}}
    {{/mapbox-map}}
</div>
```

**Comment by binhums** [comment]
Hi @GabrielCW thanks for raising. It might have something to do with the css properties here. I'll have a look and see if I can recreate.

Is there anything specific to the map / markers that you're using here?

**Comment by GabrielCW** [comment]
No, nothing in particular. I'm using quite a few jQuery plugins and libraries, are there any known collisions?

**Comment by binhums** [comment]
Not that I know of, I'll let you know if I have a solution.
